README.md

```diff
@@ -47,3 +48,11 @@ re-thinking or if you should just make a fork for yourself.
 ## If I use this it means you own my things, right?
 
 No. You are responsible for and own your own things. This code is licensed under the [BSD 3-clause license](LICENSE.md).
+
+# Financial support
```

Comment:
it should not be there?

Reply:
![image](https://user-images.githubusercontent.com/1296558/125057278-b7a7e900-e0b1-11eb-8c62-2fb66a07fe21.png)